torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1466,6 +1466,42 @@ int listReverse(Stack& stack) {
   return 0;
 }
 
+template <typename T> int minList(Stack &stack) {
```

Comment:
It would be nice if there was a way to write `minList` and `maxList` as one function, since the only difference is the comparator. But c++ 11 doesn't have partial template specialization, so i think the only way is macros which is bleh. up to you

Reply:
I think there's a way: multiplying by `-1` both sides is equivalent to switching from `<` to `>`, but I don't think it's worthwhile to deduplicate these two functions at the expense of readability.  And I hope we won't hit a bug with these, since min/max of lists is a fairly canonical problem to the point everyone just remembers it by heart.